src/mixin/FCMCustomWindow.lua

```diff
@@ -326,12 +337,24 @@ Override Changes:
 for num_args, ctrl_types in pairs({
     [0] = {"CancelButton", "OkButton",},
     [2] = {"Button", "Checkbox", "CloseButton", "DataList", "Edit", "TextEditor",
-        "ListBox", "Popup", "Slider", "Static", "Switcher", "Tree", "UpDown",
+        "ListBox", "Popup", "Slider", "Static", "Switcher", "Tree", "UpDown", "ComboBox",
     },
-    [3] = {"HorizontalLine", "VerticalLine",},
+    [3] = {"HorizontalLine", "VerticalLine", "RadioButtonGroup"},
 }) do
     for _, control_type in pairs(ctrl_types) do
-        if not finale.FCCustomWindow.__class["Create" .. control_type] then
+        local got1 = false
```

Comment:
For readability, `type_exists`? Sounds less like a nice afternoon down the river on a boat with a line and a gentle breeze...

Reply:
Fixed